web/modules/custom/bnf/src/Services/BnfImporter.php

```diff
@@ -56,7 +56,7 @@ public function getNodeTitle(string $uuid, string $endpointUrl): string {
   /**
    * Importing a node from a GraphQL source endpoint.
    */
-  public function importNode(string $uuid, string $endpointUrl, string $nodeType = 'article'): NodeInterface {
+  public function importNode(string $uuid, string $endpointUrl, string $nodeType = 'article', bool $setCanonicalSource = FALSE): NodeInterface {
```

Comment:
Well, this is pretty specific. We might want to distinguish between to/from BNF for other things too, so it would make more sense with a bool that tells if we're server or not.